### PR TITLE
fix typo in README.md

### PR DESCRIPTION
**Pull request by polferov**
Found a typo in the README.md. Thought I'd fix it :)

### CODE REVIEW

**Comment by sagiegurari** [comment]
@polferov thanks, but the readme is generated from the content.md
can you update that file as well?

**Comment by polferov** [comment]
Did do 👍. Also found the same typo in another place.

File: docs/_includes/content.md [hunk 1]
```diff
@@ -459,11 +459,11 @@ args = ["2"]
 [tasks.parallel-multi]
 run_task = { name = ["echo1", "echo2"], parallel = true }
 ```
-
+The task's current working directory value
```
Comment (sagiegurari) [inline docs/_includes/content.md:462]: i think this is by accident?

**Review by sagiegurari — CHANGES_REQUESTED** [review]

**Comment by sagiegurari** [comment]
@polferov thanks for the change and sorry for late review. i think you added one sentence by mistake. lets drop it and i'll approve the pr. thanks

**Comment by sagiegurari** [comment]
never mind, i'll merge this and remove thatline. 
thanks a lot for helping improve the docs :)

**Pull request merged**